### PR TITLE
docs(design): add autoware design overview

### PR DESCRIPTION
**Pull request by TakaHoribe**
## Description

I added a base document about the autoware software design. It was originally written in the [AutowareArchitectureProposal](https://github.com/tier4/AutowareArchitectureProposal.proj/blob/main/docs/design/software_architecture/Overview.md) and is ported here with minimum information and empty files. I'll create PRs for each component, like sensing, planning, and control. 

Please review the directory structure and the contents on the entrance page.

## Pre-review checklist for the PR author

The PR author **must** check the checkboxes below when creating the PR.

- [x] I've confirmed the [contribution guidelines].
- [x] The PR follows the [pull request guidelines].

## In-review checklist for the PR reviewers

The Reviewers **must** check the checkboxes below before approval.

- [x] The PR follows the [pull request guidelines].

## Post-review checklist for the PR author

The PR author **must** check the checkboxes below before merging.

- [ ] There are no open discussions or they are tracked via tickets.

After all checkboxes are checked, anyone who has write access can merge the PR.

[contribution guidelines]: https://autowarefoundation.github.io/autoware-documentation/main/contributing/
[pull request guidelines]: https://autowarefoundation.github.io/autoware-documentation/main/contributing/pull-request-guidelines/

### CODE REVIEW

**Comment by TakaHoribe** [comment]
@kenji-miyake I didn't even notice! Done. Thanks!

File: docs/design/autoware-architecture/.pages [hunk 1]
```diff
@@ -0,0 +1,10 @@
+nav:
+  - index.md
+  - Control
+  - Localization
+  - Perception
+  - Planning
+  - Sensing
+  - Vehicle
+  - Map
```
Comment (kenji-miyake) [inline docs/design/autoware-architecture/.pages:9]: We use `dash-case`. So you should use lower-case.
https://developers.google.com/style/filenames?hl=en

```suggestion
  - control
  - localization
  - perception
  - planning
  - sensing
  - vehicle
  - map
```

File: docs/design/autoware-architecture/.pages [hunk 2]
```diff
@@ -0,0 +1,10 @@
+nav:
+  - index.md
+  - Control
```
Comment (kenji-miyake) [inline docs/design/autoware-architecture/.pages:3]: Instead of putting empty files, I think it's good to show the page status as `under construction` like other pages.
https://github.com/autowarefoundation/autoware-documentation/blob/8c12ec64c0032a8c3bd60c3be2084a8fdf12c65f/docs/design/autoware-interfaces/index.md

image

Comment (kenji-miyake) [reply docs/design/autoware-architecture/.pages:3]: Now your page looks like this, you can't even know the title.

https://autowarefoundation.github.io/autoware-documentation/pr-86/design/autoware-architecture/Control/

File: docs/design/index.md [hunk 3]
```diff
@@ -38,6 +38,10 @@ The downside of the microautonomy architecture is that the computational perform
 
     Under Construction
 
+### Autoware architecture
```
Comment (kenji-miyake) [inline docs/design/index.md:41]: Mm, this should be at least after the `Autoware concepts` section.
And it might be placed under the `autoware-interfaces/` directory.

I'd like to ask other people about where to put these documents.

@xmfcx @mitsudome-r @isamu-takagi @LalithVipulananthan Where do you think is the best place?

File: docs/design/autoware-architecture/index.md [hunk 4]
```diff
@@ -0,0 +1,41 @@
+# Architecture Overview
+
+## Introduction
```
Comment (kenji-miyake) [inline docs/design/autoware-architecture/index.md:3]: [Nits] I think you can write this sentence here.

```suggestion
## Introduction

This page describes the architecture of Autoware.
```

File: docs/design/index.md (hunk 3 above)
Comment (isamu-takagi) [reply docs/design/index.md:41]: @kenji-miyake @TakaHoribe 
> And it might be placed under the autoware-interfaces/ directory.

I think `design/autoware-architecture/<component-name>/` is an overview and `design/autoware-interfaces/components/<component-name>/` is a detail. So the architecture focuses on what components and modules are, how they are connected, what the purpose is, and so on. On the other hand, Interface describes input and output for each components or module.

For example, in the architecture section, there are perception and planning components. And they are sending and receiving data of predicted objects to avoid or stop. Here, a predicted object is just a predicted object, it`s data structure is not explained unless it is necessary. The interface section describes that predicted object has header, uuid, classification, kinematics, etc. like [this page](https://autowarefoundation.github.io/autoware-documentation/main/design/autoware-interfaces/components/planning/#inputs).

Also, the component interface is for the public interface. The internal messages are described elsewhere, such as in the README.

Comment (kenji-miyake) [reply docs/design/index.md:41]: @isamu-takagi Thank you for your comment.
So is this good?

Autoware architecture
- target readers: beginner developers/users
- purpose: give them a high-level understanding

Autoware interfaces
- target readers: accustomed developers
- purpose: give them the information of how-to-develop

File: docs/design/autoware-architecture/index.md (hunk 4 above)
Comment (LalithVipulananthan) [reply docs/design/autoware-architecture/index.md:3]: As a general comment, I think that if a page has a title of X, then there is no need to have a sentence at the start like "This page describes X". It should be clear from the title and the section of the documentation you are in, as shown by the left sidebar.

File: docs/design/index.md (hunk 3 above)
Comment (LalithVipulananthan) [reply docs/design/index.md:41]: I think this is a good way to organise the content.  👍🏾

File: docs/design/autoware-architecture/index.md (hunk 4 above)
Comment (kenji-miyake) [reply docs/design/autoware-architecture/index.md:3]: > there is no need to have a sentence at the start like "This page describes X".

Exactly. I'd like to add a bit more additional information here to explain the scope and differences from other pages.

File: docs/design/autoware-architecture/.pages (hunk 1 above)
Comment (TakaHoribe) [reply docs/design/autoware-architecture/.pages]: fixed in 81d4bdcde01a5

File: docs/design/autoware-architecture/.pages (hunk 2 above)
Comment (TakaHoribe) [reply docs/design/autoware-architecture/.pages]: fixed in 81d4bdcde01a5

File: docs/design/autoware-architecture/index.md (hunk 4 above)
Comment (TakaHoribe) [reply docs/design/autoware-architecture/index.md:5]: fixed in 81d4bdcde01a5

File: docs/design/index.md (hunk 3 above)
Comment (TakaHoribe) [reply docs/design/index.md:48]: I updated the document. How's this?

@LalithVipulananthan I would appreciate it if you could make modifications as needed.

File: docs/design/autoware-architecture/.pages (hunk 2 above)
Comment (TakaHoribe) [reply docs/design/autoware-architecture/.pages]: modify for warning
[4620564](https://github.com/autowarefoundation/autoware-documentation/pull/86/commits/462056458ecde48490f4b3751cc37541597ff871)

File: docs/design/autoware-architecture/.pages (hunk 1 above)
Comment (kenji-miyake) [reply docs/design/autoware-architecture/.pages]: Thank you, but you should also update the directory names.
image

File: docs/design/autoware-architecture/.pages (hunk 2 above)
Comment (kenji-miyake) [reply docs/design/autoware-architecture/.pages]: Could you use `Admonitions`? :pray: 
https://squidfunk.github.io/mkdocs-material/reference/admonitions/#supported-types

Comment (TakaHoribe) [reply docs/design/autoware-architecture/.pages]: I did.
image

Comment (kenji-miyake) [reply docs/design/autoware-architecture/.pages]: Sorry, I didn't see the second commit. :bow:

File: docs/design/autoware-architecture/.pages (hunk 1 above)
Comment (TakaHoribe) [reply docs/design/autoware-architecture/.pages]: Fixed in [f979b52](https://github.com/autowarefoundation/autoware-documentation/pull/86/commits/f979b52790967555d56a4a850035c0074586287e)

File: docs/design/autoware-architecture/index.md [hunk 5]
```diff
@@ -0,0 +1,43 @@
...
+
+- Fail safe
+- HMI
+- Real-time processing
+- Redundant system
+- State monitoring system
+
+## High-level Architecture Design
+
+![Overview](image/AutowareArchitectureOverview.drawio.svg)
```
Comment (kenji-miyake) [inline docs/design/autoware-architecture/index.md:23]:
```suggestion
![Overview](image/autoware-architecture-overview.drawio.svg)
```

File: docs/design/autoware-architecture/index.md [hunk 6]
```diff
@@ -0,0 +1,43 @@
...
+
+This architecture consists of the following six stacks. Each of these design pages contains a more detailed set of requirements and use cases specific to that stack:
+
+- [Sensing Design](Sensing/index.md)
+- [Localization Design](Localization/index.md)
+- [Perception Design](Perception/index.md)
+- [Planning Design](Planning/index.md)
+- [Control Design](Control/index.md)
+- [VehicleInterface Design](Vehicle/index.md)
+- [Map Design](Map/index.md)
```
Comment (kenji-miyake) [inline docs/design/autoware-architecture/index.md:33]: [style] Use sentence case, and lower case for directories.

```suggestion
- [Sensing design](sensing/index.md)
- [Localization design](localization/index.md)
- [Perception design](perception/index.md)
- [Planning design](planning/index.md)
- [Control design](control/index.md)
- [VehicleInterface design](vehicle/index.md)
- [Map design](map/index.md)
```

Comment (TakaHoribe) [reply docs/design/autoware-architecture/index.md:33]: upps...orz sorry

Comment (TakaHoribe) [reply docs/design/autoware-architecture/index.md]: done in [f2bda3a](https://github.com/autowarefoundation/autoware-documentation/pull/86/commits/f2bda3aa760dbd6625e00c61e8a35884cfcd48e4)

File: docs/design/autoware-architecture/index.md (hunk 5 above)
Comment (TakaHoribe) [reply docs/design/autoware-architecture/index.md]: done in [f2bda3a](https://github.com/autowarefoundation/autoware-documentation/pull/86/commits/f2bda3aa760dbd6625e00c61e8a35884cfcd48e4)

File: docs/design/index.md (hunk 3 above)
Comment (LalithVipulananthan) [reply docs/design/index.md:48]: @TakaHoribe Looks fine to me!

File: docs/design/.pages [hunk 7]
```diff
@@ -1,5 +1,6 @@
 nav:
   - index.md
+  - autoware-architecture
   - autoware-concepts
```
Comment (kenji-miyake) [inline docs/design/.pages:4]: https://github.com/autowarefoundation/autoware-documentation/pull/86#discussion_r854916501

```suggestion
  - autoware-concepts
  - autoware-architecture
```

File: docs/design/autoware-architecture/.pages [hunk 8]
```diff
@@ -0,0 +1,10 @@
+nav:
+  - index.md
+  - control
+  - localization
+  - perception
+  - planning
+  - sensing
+  - vehicle
+  - map
```
Comment (kenji-miyake) [inline docs/design/autoware-architecture/.pages:9]: Did you try to order components alphabetically? :thinking: 
Or are there any reasons for putting `map` at the end?

```suggestion
  - control
  - localization
  - map
  - perception
  - planning
  - sensing
  - vehicle
```

File: docs/design/autoware-architecture/index.md [hunk 9]
```diff
@@ -0,0 +1,43 @@
...
+
+This architecture consists of the following six stacks. Each of these design pages contains a more detailed set of requirements and use cases specific to that stack:
+
+- [Sensing design](sensing/index.md)
+- [Localization design](localization/index.md)
+- [Perception design](perception/index.md)
+- [Planning design](planning/index.md)
+- [Control design](control/index.md)
+- [VehicleInterface design](vehicle/index.md)
+- [Map design](map/index.md)
```
Comment (kenji-miyake) [inline docs/design/autoware-architecture/index.md:33]: Wait for https://github.com/autowarefoundation/autoware-documentation/pull/86/files#r869025336.

File: docs/design/autoware-architecture/index.md [hunk 10]
```diff
@@ -0,0 +1,43 @@
...
+
+Note that the initial focus of this architecture design was solely on driving capability, and so the following features were left as future work:
+
+- Fail safe
+- HMI
+- Real-time processing
+- Redundant system
+- State monitoring system
+
+## High-level Architecture Design
```
Comment (kenji-miyake) [inline docs/design/autoware-architecture/index.md:21]: [style]

```suggestion
## High-level architecture design
```

File: docs/design/autoware-architecture/index.md [hunk 11]
```diff
@@ -0,0 +1,43 @@
...
+
+This page describes the architecture of Autoware.
+
+## Introduction
+
+The current Autoware is defined to be a layered architecture that clarifies each module's role and simplifies the interface between them. By doing so:
+
+- Autoware's internal processing becomes more transparent
+- Collaborative development is made easier because of the reduced interdependency between modules
+- Users can easily replace an existing module (e.g. localization) with their own software component by simply wrapping their software to fit in with Autoware's interface
```
Comment (kenji-miyake) [inline docs/design/autoware-architecture/index.md:11]: [style] I guess we need period marks here. @LalithVipulananthan Am I right?
https://developers.google.com/style/lists

```suggestion
- Autoware's internal processing becomes more transparent.
- Collaborative development is made easier because of the reduced interdependency between modules.
- Users can easily replace an existing module (e.g. localization) with their own software component by simply wrapping their software to fit in with Autoware's interface.
```

File: docs/design/autoware-architecture/.pages (hunk 8 above)
Comment (kenji-miyake) [reply docs/design/autoware-architecture/.pages:9]: Or use processing order? But I'm not sure which is the former, `localization` or `perception`.

```suggestion
  - sensing
  - map
  - localization
  - perception
  - planning
  - control
  - vehicle
```

File: docs/design/autoware-architecture/index.md [hunk 12]
```diff
@@ -0,0 +1,43 @@
...
+- [Localization design](localization/index.md)
+- [Perception design](perception/index.md)
+- [Planning design](planning/index.md)
+- [Control design](control/index.md)
+- [VehicleInterface design](vehicle/index.md)
+- [Map design](map/index.md)
+
+## Node Diagram
+
+The Autoware node diagram with the default configuration is described at the page: [node-diagram](node-diagram/index.md).
```
Comment (kenji-miyake) [inline docs/design/autoware-architecture/index.md:37]: [Question] Are you planning to send another PR to add this diagram?
https://tier4.github.io/autoware-documentation/latest/design/node-diagram/

File: docs/design/index.md [hunk 13]
```diff
@@ -40,8 +40,25 @@ The downside of the microautonomy architecture is that the computational perform
 
 ### Autoware concepts
 
+The concept of the Autoware is available [here](autoware-concepts/index.md).
+
+- Target readers:
+- Purpose:
```
Comment (kenji-miyake) [inline docs/design/index.md:46]: If it's empty, to avoid confusion:

```suggestion
TODO(TIER IV): Add target readers and purpose.
```

Comment (kenji-miyake) [reply docs/design/index.md:46]: Of course, it's better to write something.

@mitsudome-r @LalithVipulananthan Any ideas? Other sections are written as the attached image.

image

File: docs/design/autoware-architecture/index.md (hunk 12 above)
Comment (TakaHoribe) [reply docs/design/autoware-architecture/index.md:37]: Yes, right.

File: docs/design/autoware-architecture/.pages (hunk 8 above)
Comment (TakaHoribe) [reply docs/design/autoware-architecture/.pages:9]: I prefer the processing order.

File: docs/design/autoware-architecture/index.md (hunk 12 above)
Comment (TakaHoribe) [reply docs/design/autoware-architecture/index.md:37]: After this PR is merged, I'll make other PRs for 

 - node-diagram
 - planning component design
 - control component design

File: docs/design/autoware-architecture/index.md (hunk 10 above)
Comment (TakaHoribe) [reply docs/design/autoware-architecture/index.md]: Done in https://github.com/autowarefoundation/autoware-documentation/pull/86/commits/b9a3f80b26b99714f8b16cb16038677b970bc1ec

File: docs/design/autoware-architecture/control/index.md [hunk 14]
```diff
@@ -0,0 +1,5 @@
+# Control Component Design
```
[the remark this answers is not in the document]
Comment (TakaHoribe) [reply docs/design/autoware-architecture/control/index.md]: Done in https://github.com/autowarefoundation/autoware-documentation/pull/86/commits/b9a3f80b26b99714f8b16cb16038677b970bc1ec

File: docs/design/autoware-architecture/.pages (hunk 8 above)
Comment (TakaHoribe) [reply docs/design/autoware-architecture/.pages]: Done in https://github.com/autowarefoundation/autoware-documentation/pull/86/commits/b9a3f80b26b99714f8b16cb16038677b970bc1ec (for processing order)

File: docs/design/.pages (hunk 7 above)
Comment (TakaHoribe) [reply docs/design/.pages]: Done in https://github.com/autowarefoundation/autoware-documentation/commit/b9a3f80b26b99714f8b16cb16038677b970bc1ec

File: docs/design/autoware-architecture/index.md (hunk 11 above)
Comment (TakaHoribe) [reply docs/design/autoware-architecture/index.md]: follow the google style

7dcfd1b

File: docs/design/index.md [hunk 15]
```diff
@@ -40,8 +40,25 @@ The downside of the microautonomy architecture is that the computational perform
...
+
+- Target readers: all users and beginner developers
+- Purpose: describe high-level understanding
+
 ### Autoware interfaces
 
+The detailed interface design of each component are available [here](autoware-interfaces/index.md).
+
+- Target readers: intermediate developers
+- Purpose: describe the information of how-to-develop
```
Comment (kenji-miyake) [inline docs/design/index.md:60]: By the way, since this is just my comment for confirmation, I think you can delete it.
https://github.com/autowarefoundation/autoware-documentation/pull/86/files#r856877533

Instead, we can write several sentences in other PRs.

```suggestion
### Autoware interfaces

The detailed interface design of each component are available [here](autoware-interfaces/index.md).
```

File: docs/design/index.md (hunk 13 above)
Comment (kenji-miyake) [reply docs/design/index.md:46]: I think we can delete it for now.
https://github.com/autowarefoundation/autoware-documentation/pull/86/files#r869934472

Comment (kenji-miyake) [reply docs/design/index.md:46]:
```suggestion
```

File: docs/design/autoware-architecture/index.md [hunk 16]
```diff
@@ -0,0 +1,43 @@
...
+- [Planning design](planning/index.md)
+- [Control design](control/index.md)
+- [VehicleInterface design](vehicle/index.md)
+
+## Node diagram
+
+The Autoware node diagram with the default configuration is described at the page: [node-diagram](node-diagram/index.md).
+
+Note that Autoware configurations are scalable / selectable and will vary depending on the environment and required use cases.
+
```
Comment (kenji-miyake) [inline docs/design/autoware-architecture/index.md:40]: Add a link to https://autowarefoundation.github.io/autoware.universe/main/ somewhere?

**Comment by IshitaTakeshi** [comment]
How is the update on this?

File: docs/design/autoware-architecture/index.md [hunk 17]
```diff
@@ -0,0 +1,43 @@
...
+## Introduction
+
+The current Autoware is defined to be a layered architecture that clarifies each module's role and simplifies the interface between them. By doing so:
+
+- Autoware's internal processing becomes more transparent
+- Collaborative development is made easier because of the reduced interdependency between modules
+- Users can easily replace an existing module (e.g. localization) with their own software component by simply wrapping their software to fit in with Autoware's interface
+
+Note that the initial focus of this architecture design was solely on driving capability, and so the following features were left as future work:
```
[the remark this answers is not in the document]
Comment (LalithVipulananthan) [reply docs/design/autoware-architecture/index.md:13]: I think this is okay since it is an explanation of design decisions that were made in the past, and listing features that were left to be developed later on.

File: docs/design/autoware-architecture/index.md (hunk 16 above)
Comment (TakaHoribe) [reply docs/design/autoware-architecture/index.md:40]: Added. https://github.com/autowarefoundation/autoware-documentation/pull/86/commits/f5e07337b0f0406dace12d32d5bf4539b44f7673

**Comment by TakaHoribe** [comment]
Ready for merge. Review needed.

File: docs/design/index.md (hunk 13 above)
Comment (TakaHoribe) [reply docs/design/index.md]: Addressed.

File: docs/design/autoware-architecture/index.md [hunk 18]
```diff
@@ -0,0 +1,43 @@
...
+The current Autoware is defined to be a layered architecture that clarifies each module's role and simplifies the interface between them. By doing so:
+
+- Autoware's internal processing becomes more transparent.
+- Collaborative development is made easier because of the reduced interdependency between modules.
+- Users can easily replace an existing module (e.g. localization) with their own software component by simply wrapping their software to fit in with Autoware's interface.
+
+Note that the initial focus of this architecture design was solely on driving capability, and so the following features were left as future work:
+
+- Fail safe
+- HMI
```
Comment (LalithVipulananthan) [inline docs/design/autoware-architecture/index.md:16]: I think this is the first time people will come across this term, so we should expand it rather than just have an acronym.

```suggestion
- Human Machine Interface
```

File: docs/design/autoware-architecture/index.md [hunk 19]
```diff
@@ -0,0 +1,43 @@
...
+- Real-time processing
+- Redundant system
+- State monitoring system
+
+## High-level architecture design
+
+![Overview](image/autoware-architecture-overview.drawio.svg)
+
+This architecture consists of the following six stacks. Each of these design pages contains a more detailed set of requirements and use cases specific to that stack:
```
Comment (LalithVipulananthan) [inline docs/design/autoware-architecture/index.md:25]:
```suggestion
Autoware's architecture consists of the following six stacks. Each linked page contains a more detailed set of requirements and use cases specific to that stack:
```

File: docs/design/autoware-architecture/index.md [hunk 20]
```diff
@@ -0,0 +1,43 @@
...
+- [Map design](map/index.md)
+- [Localization design](localization/index.md)
+- [Perception design](perception/index.md)
+- [Planning design](planning/index.md)
+- [Control design](control/index.md)
+- [VehicleInterface design](vehicle/index.md)
+
+## Node diagram
+
+The Autoware node diagram with the default configuration is described at the page: [node-diagram](node-diagram/index.md). Detailed documents for each nodes are available in the [autoware.universe docs](https://autowarefoundation.github.io/autoware.universe/main/).
```
Comment (LalithVipulananthan) [inline docs/design/autoware-architecture/index.md:37]: Changed `autoware.universe` to `Autoware Universe` and reworded the first sentence slightly.

Does the node diagram already exist within the main Autoware documentation?

```suggestion
A diagram showing Autoware's nodes in the default configuration can be found on the [Node diagram](node-diagram/index.md) page. Detailed documents for each node are available in the [Autoware Universe docs](https://autowarefoundation.github.io/autoware.universe/main/).
```

File: docs/design/index.md [hunk 21]
```diff
@@ -40,8 +40,16 @@ The downside of the microautonomy architecture is that the computational perform
 
 ### Autoware concepts
 
+The [Autoware concepts page](autoware-concepts/index.md) describes the "design philosophy" of Autoware. Readers (service providers and all Autoware users) will know the basic concepts underlying Autoware development, such as microautonomy and core/universe architecture.
```
Comment (LalithVipulananthan) [inline docs/design/index.md:43]: I don't think there is any need for the double quotes around `design philosophy`, `overview` nor `detailed`.

```suggestion
The [Autoware concepts page](autoware-concepts/index.md) describes the design philosophy of Autoware. Readers (service providers and all Autoware users) will learn the basic concepts underlying Autoware development, such as microautonomy and the Core/Universe architecture.
```

File: docs/design/index.md [hunk 22]
```diff
@@ -40,8 +40,16 @@ The downside of the microautonomy architecture is that the computational perform
...
+
+### Autoware architecture
+
+The [Autoware architecture page](autoware-architecture/index.md) describes an "overview" of each module that composes the Autoware. Readers (all Autoware users) will know a high-level picture of how each module that composes Autoware works.
```
Comment (LalithVipulananthan) [inline docs/design/index.md:47]:
```suggestion
The [Autoware architecture page](autoware-architecture/index.md) describes an overview of each module that makes up Autoware. Readers (all Autoware users) will gain a high-level picture of how each module that composes Autoware works.
```

File: docs/design/index.md [hunk 23]
```diff
@@ -40,8 +40,16 @@ The downside of the microautonomy architecture is that the computational perform
...
+
 ### Autoware interfaces
 
+The [Autoware interfaces page](autoware-interfaces/index.md) describes a "detailed" interface of each module that composes the Autoware. Readers (intermediate developers) will know how to add a new functionality to Autoware and connect their own modules with the Autoware.
```
Comment (LalithVipulananthan) [inline docs/design/index.md:51]:
```suggestion
The [Autoware interfaces page](autoware-interfaces/index.md) describes in detail the interface of each module that makes up Autoware. Readers (intermediate developers) will learn how to add new functionality to Autoware and how to integrate their own modules with Autoware.
```

**Review by LalithVipulananthan — COMMENTED** [review]
Apologies for the late review. Please let me know if you have any comments or feedback.

**Review by kenji-miyake — COMMENTED** [review]
I think it will be okay if you fix some appearances following Lalith-san's comments.

File: docs/design/autoware-architecture/index.md (hunk 18 above)
Comment (TakaHoribe) [reply docs/design/autoware-architecture/index.md]: Thank you. Addressed.

File: docs/design/autoware-architecture/index.md (hunk 19 above)
Comment (TakaHoribe) [reply docs/design/autoware-architecture/index.md]: Thank you. Addressed.

File: docs/design/autoware-architecture/index.md (hunk 20 above)
Comment (TakaHoribe) [reply docs/design/autoware-architecture/index.md]: Thank you. Addressed.

File: docs/design/index.md (hunk 21 above)
Comment (TakaHoribe) [reply docs/design/index.md]: Thank you. Addressed.

File: docs/design/index.md (hunk 22 above)
Comment (TakaHoribe) [reply docs/design/index.md]: Addressed.

File: docs/design/index.md (hunk 23 above)
Comment (TakaHoribe) [reply docs/design/index.md]: Addressed.

**Comment by TakaHoribe** [comment]
@LalithVipulananthan Thank you for your review. I took all your suggestions in [b4ac029](https://github.com/autowarefoundation/autoware-documentation/pull/86/commits/b4ac029e70876311fe730e11e9ceaefcc9e16371).

File: docs/design/autoware-architecture/index.md [hunk 24]
```diff
@@ -0,0 +1,43 @@
...
+
+Autoware's architecture consists of the following six stacks. Each linked page contains a more detailed set of requirements and use cases specific to that stack:
+
+- [Sensing design](sensing/index.md)
+- [Map design](map/index.md)
+- [Localization design](localization/index.md)
+- [Perception design](perception/index.md)
+- [Planning design](planning/index.md)
+- [Control design](control/index.md)
+- [VehicleInterface design](vehicle/index.md)
```
Comment (LalithVipulananthan) [inline docs/design/autoware-architecture/index.md:33]:
```suggestion
- [Vehicle Interface design](vehicle/index.md)
```

Comment (LalithVipulananthan) [reply docs/design/autoware-architecture/index.md:33]: @TakaHoribe Just one last fix and then I think this PR is good to go!

**Review by LalithVipulananthan — APPROVED** [review]
LGTM!

File: docs/design/autoware-architecture/index.md (hunk 24 above)
Comment (TakaHoribe) [reply docs/design/autoware-architecture/index.md]: Thank you! Done!

(... How great if I could push just a `commit suggestion` button.)

Comment (xmfcx) [reply docs/design/autoware-architecture/index.md]: @TakaHoribe didn't the button show up for you?

Comment (TakaHoribe) [reply docs/design/autoware-architecture/index.md]: The commit with suggestion button will cause CI failure.

Comment (LalithVipulananthan) [reply docs/design/autoware-architecture/index.md]: If it's DCO that fails then if you add your sign-off (even if it's someone else's suggestion) it should pass.

Comment (TakaHoribe) [reply docs/design/autoware-architecture/index.md]: Oh really? I didn't know that sign-off works for a suggestion. I'll try it next time

Comment (TakaHoribe) [reply docs/design/autoware-architecture/index.md]: Thank you!
